src/mutations/user/addCredit.ts

```diff
@@ -8,19 +8,19 @@ import {
   TRANSACTION_PURPOSE,
 } from 'common/enums'
 import {
-  AuthenticationError,
   PaymentAmountInvalidError,
   PaymentAmountTooSmallError,
   ServerError,
+  ForbiddenError,
 } from 'common/errors'
 
 const resolver: GQLMutationResolvers['addCredit'] = async (
   _,
   { input: { amount } },
   { viewer, dataSources: { atomService, paymentService } }
 ) => {
-  if (!viewer.id) {
-    throw new AuthenticationError('visitor has no permission')
+  if (!viewer.email) {
```

Comment:
email is not required by payment services (add credit, donation, etc.), right?

Reply:
creating stripe customers need email info ?

Reply:
https://matterslab.slack.com/archives/G8877EQMS/p1692866329164109